bookstore-quarkus/src/main/kotlin/com/arconsis/http/books/BooksResource.kt

```diff
@@ -30,7 +21,6 @@ class BooksResource(private val booksDataStore: BooksDataStore) {
         return booksDataStore.getBook(id) ?: throw NotFoundException("Book with id: $id not found")
     }
 
-    @ReactiveTransactional
```

Comment:
Can you elaborate a little bit about the removeal of `@ReactiveTransactional` ?

Reply:
@ReactiveTransactional has been removed and replaced with @WithTransaction. And that one I used in the persistence service to contain the persistence related stuff in that service and not spread this domain to the rest resource.

Reply:
Yes though in more complex scenarios where you have to combine multiple methods from the repository in the service I think it makes sense to use the annotation in the service layer or in the resource.

Reply:
Yes, situations like that exist. But we should only use it in higher layers if needed to limit the scope of the transaction to what actually needs to be part of the transaction.